tests/baselines/reference/exportLocalSpellingSuggestion.js

```diff
@@ -0,0 +1,17 @@
+//// [tests/cases/conformance/es6/modules/exportLocalSpellingSuggestion.ts] ////

```

Comment:
Technically a spelling suggestion test should reside within `compiler`, not `conformance`